ldi-orchestrator/ldio-connectors/ldio-ldes-client-connector/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioLdesClientConnectorApi.java

```diff
@@ -24,7 +24,20 @@ public void handleToken(String token) {
 	public String handleTransfer(String transfer) {
 		String transer = transferService.startTransfer(transfer).getBodyAsString()
```

Comment:
This was probably out of your scope, but I discovered a type: "transer" => "transfer"

Reply:
not just a typo, a completely wrong name :stuck_out_tongue: